chain/chain/src/runtime/mod.rs

```diff
@@ -828,11 +828,31 @@ impl RuntimeAdapter for NightshadeRuntime {
     }
 
     fn get_gc_stop_height(&self, block_hash: &CryptoHash) -> BlockHeight {
+        use std::sync::atomic::{AtomicI64, Ordering};
+        use std::time::{SystemTime, UNIX_EPOCH};
+
+        // Use AtomicI64 to store unix timestamp in seconds
+        static LAST_LOG_TIME: AtomicI64 = AtomicI64::new(0);
```

Comment:
I am not happy with global state, I would rather this was defined inside the `self` however that will require interior mutability and be a more complicated change.  So we can keep this as is.